api/main.py

```diff
@@ -121,22 +122,23 @@ async def get_segments_for_file(
 
     try:
         database = get_db()
-        segments_query = database.AQLQuery(
-            query=main_queries.QUERY_FILE_SEGMENTS_PARALLELS,
+        table_query = database.AQLQuery(
+            query=main_queries.QUERY_TABLE_VIEW,
             batchSize=10000,
             bindVars={
                 "filename": file_name,
                 "score": score,
                 "parlength": par_length,
                 "coocc": co_occ,
+                "sortkey": "parallels_sorted_by_src_pos",
                 "limitcollection_positive": limitcollection_positive,
                 "limitcollection_negative": limitcollection_negative,
                 "page": page,
```

Comment:
This will need a 
`"start_folio": get_folio_regex(language, file_name, folio),`
added